packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -330,7 +331,7 @@ export function fallbackApplePalette(): ThemeColorDefinition {
     inputPlaceholderText: applePlatform.placeholderTextColor,
 
     // Set the default button tokens to match the Acrylic Button style
-    buttonBackground: fluentUIApple.neutralBackground3,
+    buttonBackground: fluentUIApple.neutralBackground1,
```

Comment:
We might be able to remove the comment on line 333. When I made the FURN macOS button, I copied the design from the FluentUI Apple macOS button. Now that we may be deviating from that and using design tokens instead, if the comment is not relevant we should remove it.

Reply:
Oh... actually I tried to match it to the Acrylic style too so maybe worth keeping? also @harrieshin suggested to follow the macOS Button spec (without design tokens) to style the default button

Reply:
Oh, if it's still the acrylic style than nvm. Comment still useful!